### PR TITLE
Works and about section

### PR DESCRIPTION
**Pull request by JhonatanErosa**
## Works & About Me Section

* Add works section-mobile version
* Add About Me section
* Fix linters in the CSS file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,5 +49,148 @@ <h2>I'm a Software Developer</h2>
...
+                <div class="cardContent2">
+                    <h2>Data Dashboard Healthcare</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="categories">
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="projectsButton2">See project</a>
```
Comment (tufoinnkuo10) [inline index.html:132]: - [x] Please kindly implement the button hover, and pressed functionalities like displayed in the Figma design. please add for all your buttons. 

Screenshot (1214)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @JhonatanErosa 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (JhonatanErosa) [reply index.html:132]: Excellent suggestion 👍

**Review by Cabraham1 — APPROVED** [review]
Hi @JhonatanErosa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**